app/Shell/DockerTags.php

```diff
@@ -44,7 +44,12 @@ public function getTags(): Collection
     {
         $response = json_decode($this->getTagsResponse()->getContents(), true);
 
-        return collect($response['results'])->map->name->filter();
+        return collect($response['results'])
```

Comment:
I'm a little nervous about sorting it ourselves; we may miss some Docker-specific versioning constraints that we're not aware of yet.

Was it not possible to pass the query parameter in that I mentioned?

Reply:
That did not sort properly, unfortunately. If you look here: https://registry.hub.docker.com/v2/repositories/library/postgres/tags?order=name you will see version `13.2` is not at the top nor the bottom.

Reply:
I also tried `ordering=name` with the same result. If you look in #207 you'll see the list that comes back isn't in the proper order.

Reply:
Ahh, bummer. OK--let's push this into `main` and then some folks can test it out before we tag it.

Thanks!